### ISSUE TITLE
KeyboardState contains Keys.None for some keys

### ISSUE THREAD

**Issue by theranix** [FNA-XNA/FNA#411]
FNA_KEYBOARD_USE_SCANCODES is false

- When en keyboard is used pressing keys "[];',./" works - everything i ok

```
mod: SDL2.SDL.SDL_Keymod.KMOD_NONE
scancode: SDL2.SDL.SDL_Scancode.SDL_SCANCODE_COMMA
sym: SDL2.SDL.SDL_Keycode.SDLK_COMMA
```

- When cyrillic keyboard is used pressing the same keys doesn't work

```
mod: SDL2.SDL.SDL_Keymod.KMOD_NONE
scancode: SDL2.SDL.SDL_Scancode.SDL_SCANCODE_COMMA
sym: 1073
```

Will it be right solution if I fix it by adding new records to INTERNAL_keyMap?

**Comment by flibitijibibo** [comment]
If SDL recognizes them, we can add them to our dictionary as well - if you happen to know the char as well that will be helpful too, since the integer value is sometimes hard to parse when looking things up in the future (you'll see examples in SDL2_FNAPlatform).

**Comment by 7aGiven** [comment]
@flibitijibibo This issue can be closed. SDL3 add a hint that SDL_HINT_KEYCODE_OPTIONS. It's default value is "french_numbers,latin_letters".

> "latin_letters": For keyboards using non-Latin letters, such as Russian
  or Thai, the letter keys generate keycodes as though it had an English
  QWERTY layout. e.g. pressing the key associated with SDL_SCANCODE_A on a
  Russian keyboard would yield 'a' instead of a Cyrillic letter.
